frontend_build/src/read_bundle_plugins.js

```diff
@@ -27,6 +27,7 @@ var libs = function(kolibri_name) {
     'vuex': kolibri_name + '.lib.vuex',
     'core-base': kolibri_name + '.lib.coreBase',
```

Comment:
Bah, I missed deleting these three lines. Could you remove them, confirm that the build still works (after a restart) and push again?